python/tvm/relay/module.py

```diff
@@ -157,5 +157,26 @@ def get_global_type_var(self, name):
         return _module.Module_GetGlobalTypeVar(self, name)
 
     @staticmethod
-    def from_expr(expr):
-        return _module.Module_FromExpr(expr)
+    def from_expr(expr, functions=None, type_defs=None):
+        """Construct a module from a standalone expression.
+
+        Parameters
+        ----------
+        expr: Expr
+            The starting expression
+        global_funcs: Optional[dict]
+            Map of global vars to function definitions
+        type_defs: Optional[dict]
+            Map of global type vars to type definitions
+
+
+        Returns
+        -------
+        mod: Module
+            A module containing the passed definitions,
+            where expr is set as the entry point
+            (wrapped in a function if necessary)
+        """
+        return _module.Module_FromExpr(expr,
+                                       functions if functions is not None else {},
```

Comment:
Can you maybe lift this out? I don't really like inline conditions like this, they are hard to modify/debug.